packages/blade/src/components/Input/PasswordInput/PasswordInput.stories.tsx

```diff
@@ -208,4 +211,49 @@ ControlledInput.parameters = {
   },
 };
 
+const PasswordInputHookFormTemplate: ComponentStory<typeof PasswordInput> = () => {
+  const { control, handleSubmit } = useForm<{ password: string }>({
+    defaultValues: { password: '' },
+  });
+
+  const { field, fieldState } = useController({
```

Comment:
this we should not add to stories but add to integrations like @divyanshu013 had added. `withReactHookForm`

Reply:
you mean added these to /examples folder?

Reply:
done.